lib/linguist/languages.yml

```diff
@@ -1659,13 +1659,16 @@ HTML+Django:
   tm_scope: text.html.django
   group: HTML
   extensions:
-  - ".mustache"
   - ".jinja"
+  - ".mustache"
```

Comment:
To make it alphabetical. The tests told me off since i put `njk` before `jinja`, so i just sorted the whole thing 

https://travis-ci.org/github/linguist/jobs/178098254 (line 1108)